apps/etterlatte-behandling/src/main/kotlin/behandling/etteroppgjoer/brev/EtteroppgjoerRevurderingBrevService.kt

```diff
@@ -154,7 +154,12 @@ class EtteroppgjoerRevurderingBrevService(
                         faktiskInntekt = Kroner(beregnetEtteroppgjoerResultat.nyBruttoStoenad.toInt()),
                         grunnlag = EtteroppgjoerBrevGrunnlag.fra(faktiskInntekt),
                     ),
-                brevRedigerbarInnholdData = EtteroppgjoerBrevData.VedtakInnhold(),
+                brevRedigerbarInnholdData =
+                    EtteroppgjoerBrevData.VedtakInnhold(
+                        etteroppgjoersAar = detaljertForbehandling.behandling.aar,
+                        forhaandsvarselSendtDato = detaljertForbehandling.behandling.opprettet.toLocalDate(), // TODO: annen dato?
```

Comment:
Jeg tror det beste er å hente distribusjonsdato for brevet i forbehandling her. Må sikkert kodes opp.